### ISSUE TITLE
[sandbox-htmlize] CBOR diagnostic is center-aligned

### ISSUE THREAD

**Issue by thomas-fossati** [ietf-tools/datatracker#4494]
### Describe the issue

Please compare the rendering of Figure 3 in
* https://sandbox-htmlize.ietf.org/doc/html/draft-ietf-core-problem-details-08#name-custom-problem-detail-entri

against

* https://datatracker.ietf.org/doc/html/draft-ietf-core-problem-details-08#section-3.2

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by larseggert** [comment]
I think this is an issue with the CSS - @martinthomson?

**Comment by martinthomson** [comment]
Indeed it is.  The outer div is marked with the alignCenter class, but I believe that the intent was to center the block of code, not each line of code.

**Comment by martinthomson** [comment]
OK, something odd is going on.  The code I provided shouldn't have had this problem, because I don't include a `text-align: center` directive for `.alignCenter` and yet it is in the stylesheet.  In looking at this more closely, I've made a few tweaks in <https://github.com/martinthomson/rfc-txt-html/commit/aed63bcf14490d30b6338c28a91420bfd65e42d8>.

(Lars, I also have a fix that might help with the printing of the authors block, but that's on a computer in a different country.  Remind me to get you a copy.)
